app/lib/CaptureRecorder.js

```diff
@@ -18,7 +20,7 @@ function CaptureRecorder(app, captureName) {
   assert(captureName, 'Capture name required');
 
   this.app = app;
-  this.capturePath = path.normalize(app.config.capturesDir + '/' + captureName);
+  this.capturePath = path.normalize(app.config.capturesDir + '/' + captureName); // eslint-disable-line prefer-template
```

Comment:
Could just go with a template string here.

Reply:
Definitely could. I can go back through and make those happen.